gallery/src/components/NxRadio/NxRadioPage.tsx

```diff
@@ -107,6 +107,15 @@ const NxRadioPage = () =>
               multiple overlapping tooltips from appearing.
             </NxTable.Cell>
           </NxTable.Row>
+          <NxTable.Row>
```

Comment:
`NxRadio` doesn't have this prop.  It's entire radio groups (e.g. `NxFieldset`s) that may be required, not individual radios